SearchAPI/CMR/Translate/parse_cmr_response.py

```diff
@@ -68,7 +68,7 @@ def remove_field(f):
         remove_field('platform')
 
     if 'frameNumber' in req_fields:
-        asf_frame_platforms = ['Sentinel-1A', 'Sentinel-1B', 'ALOS']
+        asf_frame_platforms = ['Sentinel-1A', 'Sentinel-1B', 'ALOS', 'SENTINEL-1A', 'SENTINEL-1B']
         result['frameNumber'] = get_val(attr_path('FRAME_NUMBER')) \
             if result['platform'] in asf_frame_platforms \
             else get_val(attr_path('CENTER_ESA_FRAME'))
```

Comment:
Technically we could do something like his maybe?

```python
asf_frame_platforms = [''ALOS', 'SENTINEL-1A', 'SENTINEL-1B']
if result['platform'].upper() in asf_frame_platforms:
    pass
```
But tbh since it's SearchAPI and we're re-writing it soon anyways, it's not worth blocking at all